### ISSUE TITLE
Data leakage for global univariate setting?

### ISSUE THREAD

**Issue by gorold** [rakshitha123/TSForecasting#13]
Hi, thanks for your amazing work! One issue I'm wondering about is, what constitutes a data leak in the global univariate setting?

Please correct me if I'm wrong, but I noticed that for the global univariate settings where the time series is unaligned (specifically, an unaligned end date),  this leads to a shorter time series X to have test set in the same period as a longer time series Y's training set. (Examples would include Vehicle Trips and Tourism datasets). 

There could possibly be some leakage of global information (e.g. in some financial time series there could be some systematic movement of all assets), leading to unfair advantage for global models.

**Comment by cbergmeir** [comment]
Hi, this is a very good observation. We actually discuss this issue in a recent paper we have, in its "Data Leakage" section, here (it is open access):  

https://doi.org/10.1007/s10618-022-00894-5 

The issue has also been looked at here: 

https://thiyanga.netlify.app/talk/isf20-talk/

Unfortunately there is not much to be done about it, as far as I can think of. Only to take results of global versus local models on M3, M4 with a grain of salt. Anyway, global models have also shown in many aligned use cases their superiority (e.g., M5)

**Comment by gorold** [comment]
Thanks a lot, this is really helpful!
